CONTRIBUTING.md

```diff
@@ -172,19 +173,38 @@ Describe a new checker in [checkers section](./README.md#checkers).
 ### error-compare
 
 ```go
-❌   assert.ErrorContains(t, err, "not found")
-     assert.EqualError(t, err, "user not found")
+❌   assert.Contains(t, err.Error(), "not found")
      assert.Equal(t, err.Error(), "user not found")
      assert.Equal(t, err, errSentinel) // Through `reflect.DeepEqual` causes error strings to be compared.
      assert.NotEqual(t, err, errSentinel)
      // etc.
 
+✅   assert.ErrorContains(t, err, "not found")
+     assert.EqualError(t, err, "user not found")
+     assert.ErrorIs(t, err, ErrUserNotFound)
+     assert.NotErrorIs(t, err, errSentinel)
+```
+
+**Autofix**: true. <br>
+**Enabled by default**: true. <br>
+**Reason**: Using testify methods will avoid the risk of nil pointer exception that might happen with the use of the `Error()` method on the `error` interface. <br>
+**Related issues**: [#47](https://github.com/Antonboom/testifylint/issues/47)
+
+---
+
+### error-contains
+
+```go
+❌   assert.ErrorContains(t, err, "not found")
+
+
 ✅   assert.ErrorIs(t, err, ErrUserNotFound)
+     assert.ErrorAs(t, err, &target)
 ```
 
 **Autofix**: false. <br>
 **Enabled by default**: true. <br>
-**Reason**: The `Error()` method on the `error` interface exists for humans, not code. <br>
+**Reason**: The `assert.ErrorContains()` is an anti-pattern. The test needs to be adapted to use `assert.ErrorIs()` or `assert.ErrorAs()` methods. <br>
```

Comment:
I still disagree it's a bad pattern when used to check for example a value is provided to sprintf, like 

```go
fmt.Errorf("%w: %d", ErrUnexpectedStatusCode, res.Status)
```

+  

```go
assert.ErrorIs(t, err, ErrUnexpectedStatusCode)
assert.ErrorContains(t, err, "404")
```

But I know most people are simply not using the right way.


```suggestion
**Reason**: The `assert.ErrorContains()` is commonly used as an anti-pattern. The code and tests often needs to be adapted to be able to use `assert.ErrorIs()` or `assert.ErrorAs()` methods. <br>
```